packages/components/src/mobile/bottom-sheet/radio-cell.native.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, check } from '@wordpress/icons';
+import { usePreferredColorSchemeStyle } from '@wordpress/compose';
+/**
+ * Internal dependencies
+ */
+import Cell from './cell';
+import styles from './styles.scss';
+
+export default function BottomSheetColorCell( props ) {
+	const { selected, ...cellProps } = props;
+
+	const selectedIconStyle = usePreferredColorSchemeStyle(
+		styles.selectedIcon,
+		styles.selectedIconDark
+	);
+
+	return (
+		<Cell
+			{ ...cellProps }
+			accessibilityRole={ 'radio' }
+			accessibilityState={ { selected } }
+			accessibilityHint={
+				/* translators: accessibility text (hint for selecting option) */
+				__( 'Double tap to select the option' )
+			}
+			editable={ false }
+			value={ '' }
```

Comment:
Do we need to pass an empty value here?

Reply:
Yes, we need that to _avoid_ centred label

Reply:
Maybe it is worth to check if we could fix it in the Cell instead of passing the empty string as a value :)